### DISCUSSION TITLE
Firmware version 16.3.x works the best in terms of xDSL sync and compatibility. Use if available

### DISCUSSION THREAD

**Discussion by Sam-arch99** [hack-technicolor/hack-technicolor#107] [General]
What xDSL driver version is used in 16.3.x? It's not mentioned on your website...

Check Version: xdslctl --version

**Comment by LuKePicci** [comment]
It really depends on the particular firmware release date, the major number does not say much about that. That indication comes from original Telstra-centric documentation and was referring to VANT-F firmwares. Different boards have different DSL chips so no actual comparison makes sense. Just use the latest firmware, 16.x are too old and lack many other features.

We should probably drop that suggestion, if you really need to switch DSL drivers you can do it on the latest firmwares as well, the custom tch-nginx-gui allows it in a user-friendly fashion.

**Comment by Sam-arch99** [comment]
Gotya. I've tried many different xDSL drivers with my DGA4132 but there is not that much difference on my Vectoring VDSL connection (17a). I'm currently using a pretty new driver (A2pvfbH045q) but I do think it was made/optimized for g.fast products, anyway it does work fine with my connection.
